cmd/openfga/main_test.go

```diff
@@ -577,27 +576,84 @@ func GRPCCheckTest(t *testing.T, tester OpenFGATester) {
 				},
 			},
 			testData: &testData{
-				typeDefinitions: []*openfgapb.TypeDefinition{
-					{
-						Type: "repo",
-						Relations: map[string]*openfgapb.Userset{
-							"admin": {
-								Userset: &openfgapb.Userset_Union{
-									Union: &openfgapb.Usersets{Child: []*openfgapb.Userset{
-										{
-											Userset: &openfgapb.Userset_This{
-												This: &openfgapb.DirectUserset{},
-											},
-										},
-									}},
-								},
+				model: &openfgapb.AuthorizationModel{
+					TypeDefinitions: []*openfgapb.TypeDefinition{
+						{
+							Type: "repo",
+							Relations: map[string]*openfgapb.Userset{
+								"admin": typesystem.Union(typesystem.This()),
 							},
 						},
 					},
 				},
-				tuples: &openfgapb.TupleKeys{
-					TupleKeys: []*openfgapb.TupleKey{
-						tuple.NewTupleKey("repo:auth0/express-jwt", "admin", "github|bob@auth0.com"),
+				tuples: []*openfgapb.TupleKey{
+					tuple.NewTupleKey("repo:auth0/express-jwt", "admin", "github|bob@auth0.com"),
```

Comment:
Can we drop the `@auth0.com` from this, and in the future all, examples? We shouldn't encourage others to write tuples with these special characters 😱

Reply:
Resolving since this will be resolved in rebase